web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -161,6 +163,9 @@
   {/if}
 
   <DetailPanelDescription {asset} {isOwner} />
+  {#if $preferences?.app_settings?.rating}
+    <DetailPanelRating {asset} {isOwner} />
```

Comment:
This won't be visible for shared links

Reply:
Yes, I think that is ok? Just added !isSharedLink() to the if statement for safety